src/server/resources/venue-queries.yaml

```diff
@@ -1,4 +1,4 @@
-retrieveVenues: >
+readVenues: >
```

Comment:
I am using the fetchr method names: https://github.com/yahoo/fluxible-plugin-fetchr#actioncontext